packages/core/admin/admin/src/pages/MarketplacePage/tests/index.test.js

```diff
@@ -90,9 +124,12 @@ describe('Marketplace page', () => {
     expect(trackUsage).toHaveBeenCalledTimes(1);
   });
 
-  it('should return plugin search results matching the query', async () => {
+  it('should return plugin search results matching the query', () => {
     const { container } = render(App);
-    const input = await getByPlaceholderText(container, 'Search');
+    const pluginsTab = screen.getByRole('tab', { name: /plugins/i });
+    fireEvent.click(pluginsTab);
```

Comment:
Why is this necessary now?

Reply:
Related to my comment above, it was about making sure the tests work in isolation, but it's not the right approach